### PR TITLE
Documantation changes to EDL

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46470

## Description
Fixing documanation for EDL
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/README.md [hunk 1]
```diff
@@ -85,22 +94,22 @@ To access the Export Indicators service by instance name, make sure ***Instance
...
+| ca                | Only with `proxysg` format. The categories which are exported. Indicators not falling into these categories are classified as the default category.                 | `https://{server_host}/instance/execute/{instance_name}?v=proxysg&ca=category1,category2`           |
+| tx                | Whether to output `CSV` format as textual web pages.                                                                                                                | `https://{server_host}/instance/execute/{instance_name}?v=CSV&tx`                                   |
+| mc                | Configure max CIDR size.                                                                                                                                            | `https://{server_host}/instance/execute/{instance_name}?mc=10`                                      |
+| nt                 | Configure if rxclude top level domainGlobs.                                                                                                                         | `https://{server_host}/instance/execute/{instance_name}?nt=true`                                    |
```
Comment (ytauman) [inline Packs/EDL/Integrations/EDL/README.md:112]: typo

File: Packs/EDL/Integrations/EDL/README.md [hunk 2]
```diff
@@ -72,7 +72,16 @@ These parameters prevent the integration from incorrectly inserting unwanted TLD
 The default value for `Maximum CIDR network prefix bits size` is 8, which means that CIDRs with a lower network prefix bits number are not included (such as 0.0.0.0/2).
 
 
-The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as \*.com, \*.co.uk, \*.org and other top level domains.
+The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as `*.com`, `*.co.uk`, `*.org` and other top level domains.
+
+### DomainGlobs
+
+When parsing DomainGlobs the parser will create two different inputs as that is usually how DNS Firewalls work. For example if the DomainGlob `*.bad.com` will be parsed it will output two lines to the list:
```
Comment (ytauman) [inline Packs/EDL/Integrations/EDL/README.md:79]: looks like the whole document refers to this as domainGlobs (small d) so i think it's best to keep that aligned here as well

File: Packs/EDL/Integrations/EDL/README.md [hunk 3]
```diff
@@ -72,7 +72,16 @@ These parameters prevent the integration from incorrectly inserting unwanted TLD
 The default value for `Maximum CIDR network prefix bits size` is 8, which means that CIDRs with a lower network prefix bits number are not included (such as 0.0.0.0/2).
 
 
-The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as \*.com, \*.co.uk, \*.org and other top level domains.
+The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as `*.com`, `*.co.uk`, `*.org` and other top level domains.
+
+### DomainGlobs
```
Comment (ytauman) [inline Packs/EDL/Integrations/EDL/README.md:77]: i'm not sure if we should name the section like this.
maybe 'Behaviors' or something like that.
@Ni-Knight wdyt? so it's safeguards and then behaviors..?

image

File: Packs/EDL/Integrations/EDL/README.md (hunk 1 above)
Comment (ytauman) [reply Packs/EDL/Integrations/EDL/README.md:112]: Configure if **to** exclude top level domainGlobs.

File: Packs/EDL/Integrations/EDL/README.md (hunk 2 above)
Comment (ytauman) [reply Packs/EDL/Integrations/EDL/README.md:79]: also - i might say 'when parsing domainGlob indicator types' - to make it explicit that it's a type-related behavior.

Comment (Ni-Knight) [reply Packs/EDL/Integrations/EDL/README.md:79]: capital D is my bad there.

File: Packs/EDL/Integrations/EDL/README.md (hunk 3 above)
Comment (Ni-Knight) [reply Packs/EDL/Integrations/EDL/README.md:77]: maybe?
### Unique Behaviors
#### DomainGlobs
Text Text Text
#### IP Ranges
Text Text Text
```
### Unique Behaviors
#### DomainGlobs
Text Text Text
#### IP Ranges
Text Text Text
```

Comment (ilaner) [reply Packs/EDL/Integrations/EDL/README.md:77]: The `IP Ranges` and the `Drop Invalids` are already documented in the parameters/arguments section

**Review by richardbluestone — APPROVED** [review]

**Pull request merged**